.travis.yml

```diff
@@ -34,16 +34,10 @@ env:
     - TEST_AIIDA_BACKEND=django COMPILE_DOCS=true
 
 install:
-    # This is required for some recent features of setup.py
+    # Upgrade pip setuptools and wheel to be able to run the next command
     - pip install -U pip wheel setuptools
-
     # Install AiiDA with some optional dependencies
-    - pip install -U .[verdi_shell,ssh,REST] --process-dependency-links
-
-    # This should go away, put requirements (also optional) in setup.py
-    # But for the time being, we keep it (needed to avoid skipping tests
-    # for missing dependencies)
-    - pip install -r optional_requirements.txt
+    - pip install .[verdi_shell,ssh,REST,docs,atomic_tools] --process-dependency-links
```

Comment:
I thought exactly the same, but it turned out to be not true when I tested it. Let me make an example. In my environment pip is installed to version 9.0.1. Then I downgrade:
```bash
pip install pip==8.0.1
...
Installing collected packages: pip
  Found existing installation: pip 9.0.1
    Uninstalling pip-9.0.1:
      Successfully uninstalled pip-9.0.1
Successfully installed pip-8.0.1
```
it works perfectly fine without need for `-U` option.
Now in my understanding  `-U`  only serves to let the system look for the latest available version of a certain package and install it.